src/files/tables.py

```diff
@@ -15,3 +17,8 @@ class Meta:
         model = BaseFile
         template_name = "django_tables2/bootstrap.html"
         fields = ("uuid", "albums", "attribution", "uploader", "license", "file_size", "status", "title")
+
+    def render_uuid(self, value: str) -> str:
```

Comment:
is this in use anywhere?

Reply:
Yes, it's for getting anchor tags with a href back to the "/files/<pk>" detail page inside the django-table.